lib/application/view-models/home/update_collection_vm.dart

```diff
@@ -129,10 +137,13 @@ class CollectionMetadata extends Equatable {
 class MemoMetadata extends Equatable {
   const MemoMetadata({required this.question, required this.answer});
 
-  factory MemoMetadata.empty() => const MemoMetadata(question: '', answer: '');
+  factory MemoMetadata.empty() => const MemoMetadata(question: RichTextEditingValue(), answer: RichTextEditingValue());
+
+  MemoMetadata copyWith({RichTextEditingValue? question, RichTextEditingValue? answer}) =>
+      MemoMetadata(question: question ?? this.question, answer: answer ?? this.answer);
 
-  final String question;
-  final String answer;
+  final RichTextEditingValue question;
```

Comment:
This is leaking a "UI" value into the VM

Reply:
I've added a new entity named `MemoUpdateContent`, which holds the rich and plain text versions of the question/answer. Let me know what do you think.